### PR TITLE
[aptos-ledger] Added readme for publishing

### PR DESCRIPTION
**Pull request by 0xjinn**
### Description

1. Added a readme with some example
2. Update the toml to point to correct repo on github

### CODE REVIEW

File: crates/aptos-ledger/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Examples
+
+### Get Public Key from your Aptos account on Ledger
+
+```rust
+use aptos_ledger::get_public_key;
+
+let public_key = match get_public_key(false);
+println!("Version: {:?}", version);
```
Comment (0xmaayan) [inline crates/aptos-ledger/README.md:19]: where is the `version` coming from?

File: crates/aptos-ledger/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# aptos-ledger
+
+Provides a set of commands and methods to communicate with Aptos app on Ledger
+
+- Read the Aptos app official name and current version
```
Comment (0xmaayan) [inline crates/aptos-ledger/README.md:5]:
```suggestion
The methods supported by the aptos-ledger are:
- Read the Aptos app official name and current version
```
or something like that

File: crates/aptos-ledger/README.md (hunk 1 above)
Comment (0xjinn) [reply crates/aptos-ledger/README.md:19]: oops! Good catch

File: crates/aptos-ledger/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## Examples
+
+### Get Public Key from your Aptos account on Ledger
+
+```rust
+use aptos_ledger::get_public_key;
+
+let public_key = match get_public_key(false);
+println!("Public Key: {:?}", version);
```
Comment (0xmaayan) [inline crates/aptos-ledger/README.md:22]: should it be

```suggestion
println!("Public Key: {:?}", public_key);
```

File: crates/aptos-ledger/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# aptos-ledger
+
+Provides a set of commands and methods to communicate with Aptos app on Ledger
+
+The methods supported by the aptos-ledger are:
+
+- get_app_version()
```
Comment (banool) [inline crates/aptos-ledger/README.md:7]: Nit:
```suggestion
- `get_app_version()`
```

**Review by banool — APPROVED** [review]

**Review by 0xmaayan — APPROVED** [review]

File: crates/aptos-ledger/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# aptos-ledger
+
+Provides a set of commands and methods to communicate with Aptos app on Ledger
+
+The methods supported by the aptos-ledger are:
+
+- `get_app_version()`
+- get_app_name()
```
Comment (0xmaayan) [inline crates/aptos-ledger/README.md:8]:
```suggestion
- `get_app_name()`
- `get_public_key()`
- `sign_txn()`
```

**Pull request merged**